Daf.Core.VSCode/src/extension.ts

```diff
@@ -224,6 +238,15 @@ async function createUserFile(): Promise<void>
 	});
 }
 
+async function reloadPlugins(): Promise<void>
+{
+	if (lspClient != undefined){
+		lspClient.sendRequest('daf/reloadplugins');
+	} else {
+		vscode.window.showInformationMessage('Cannot reload Daf plugins, language server has not been initiated.');
+	}
```

Comment:
For the most part it appears that we put braces on newlines in this document, but not always.. I wonder if we should decide on a standard and stick to it? I know it's more common to put it in the same brace in typescript, so maybe we should stick with that. It's just so awful to look at...

Reply:
I'd rate following the most common way of writing it higher than personal preference, though this seems to be the only place where it's written at the same line. I'll change it to be similar to the other code in the file for now, then we can discuss how we should format it in a separate issue.